docs/content/guides/tools-and-building/custom-builds.md

```diff
@@ -75,8 +75,8 @@ Each Handsontable [project](#about-building) has its own building processes defi
 ## Running your first build
 
 To run your first build:
-1. Install [Node.js](https://nodejs.org/) (version **16.14**+).
-2. Install [npm](https://www.npmjs.com/) (version **8.5**+).
+1. Install [Node.js](https://nodejs.org/) (version **16.17**+).
+2. Install [npm](https://www.npmjs.com/) (version **8.15**+).
```

Comment:
This is being discussed internally here: https://handsoncode.slack.com/archives/C4PV4S33Q/p1662975119755469

Reply:
We decided in the above-linked thread to update the versions before a release, not here. I will revert these lines.